presto-bytecode/src/main/java/com/facebook/presto/bytecode/BytecodeUtils.java

```diff
@@ -85,7 +85,12 @@ private Bootstrap() {}
         public static CallSite bootstrap(MethodHandles.Lookup callerLookup, String name, MethodType type, long bindingId)
         {
             ClassLoader classLoader = callerLookup.lookupClass().getClassLoader();
-            checkArgument(classLoader instanceof DynamicClassLoader, "Expected %s's classloader to be of type %s", callerLookup.lookupClass().getName(), DynamicClassLoader.class.getName());
+            checkArgument(
+                    classLoader instanceof DynamicClassLoader,
+                    "Expected %s's classloader to be of type %s, but have %s",
+                    callerLookup.lookupClass().getName(),
+                    DynamicClassLoader.class.getName(),
+                    classLoader.getClass().getName());
```

Comment:
may not be necessary - however - is there a possible that `classLoader` may be null ?